content/en/users/workload-manager/_index.md

```diff
@@ -4,9 +4,463 @@ linkTitle: "Workload Manager"
+* *Log In options*: the service supports both X.509, Certificate and Check-in
+log-in.
+* *View options*: allow to choose either desktop or tabs layout.
+* *Menu*: a list of tools that enable the selected VO.
+
+#### Upload Proxy
+
+Before submitting your job, you need to upload your Proxy.  Login to the portal.
+Go to:  
+`Menu` > `Tools` > `Proxy Upload`, enter your certificates .p12 file and
+the passphrase, click `Upload`.
+
+![Fig. 3 - The wizard to upload the .p12 proxy certificate](./figure_3.png)
+![Fig. 3.1 - The wizard to upload the .p12 proxy certificate](./figure_3_1.png)
+
+Fig. 3 - The wizard to upload the .p12 proxy certificate
+
+#### Job Submission
+
+Go to:  
+`Menu` > `Tools` > `Job Launchpad`.  First check the `Proxy Status`, click it
+until it shows `Valid` in green color.
+
+In the Job Launchpad, you can select your jobs from the list; add parameters,
+indicating the output Sandbox location.
+
+Now, select `Helloworld` from the  job list, and click `Submit`, you just launch
+your very first job to the EGI HTC cluster.
+
+![Fig. 4 - Submit a job with the Job Launchpad](./figure_4.png)
+
+Fig. 4 - Submit a job with the Job Launchpad
+
+#### Monitor Job status
+
+Go to:  
+`Menu` > `Applications` > `Job Monitor`.  
+The left panel gives all kinds of search options for your jobs. Set your search
+criteria, and click `Submit`, the jobs will list on the right panel.  
+Try the various options to view different information about the jobs.
+
+![Fig. 5 - Monitor the job execution with the Job Monitor panel](./figure_5.png)
+
+Fig. 5 - Monitor the job execution with the Job Monitor panel
+
+#### Get Results from Sandbox
+
+Once the job has been successfully processed, the `Status` of the job will
+change to green.  Right click the job, select:  
+`Sandbox` > `Get Output file(s)`,
+you can get the result file(s).
+
+#### Full User Guide for DIRAC Web Portal
+
+For further instructions, please refer to
+[DIRAC Web Portal Guide](https://dirac.readthedocs.io/en/latest/UserGuide/WebAppUserGuide/index.html)
+
+### The DIRAC client tool
+
+The easiest way to install the client is via
+[Docker Container](https://www.docker.com/).
+If you have a Docker client installed in your machine, install the DIRAC CLI
+as follows:
+
+```sh
+$ docker run -it -v $HOME:$HOME -e HOME=$HOME diracgrid/client:egi
+```
+
+Once the client software is installed, it should be configured in order to
+access the EGI Workload Manager service:
+
+```sh
+$ source /opt/dirac/bashrc
+```
+
+To proceed further a temporary proxy of the user certificate should be created.
+This is necessary to get information from the central Configuration Service:
+
+```sh
+$ dirac-proxy-init -x
+Generating proxy...
+Enter Certificate password:
+...
+```
+
+Now the client can be configured to work in conjunction with the EGI Workload
+Manager service:
+
+```sh
+$ dirac-configure defaults-egi.cfg Executing:
+/home/larocca/DIRAC/DIRAC/Core/scripts/dirac-configure.py defaults-egi.cfg
```

Comment:
beware of the `larocca` path here

Reply:
I was wondering if we shouldn't anonymise/make more generic all the part with info about @glarocca (cert and so on).
Like by using a `Jane Doe` and non-real data?